dotnet/src/Microsoft.Agents.AI.Workflows.Declarative/CodeGen/AddConversationMessageTemplate.tt

```diff
@@ -16,15 +16,17 @@ internal sealed class <#= this.Name #>Executor(FormulaSession session, WorkflowA
         {
             throw new DeclarativeActionException($"Conversation identifier must be defined: {this.Id}");
         }
-        ChatMessage newMessage = new(ChatRole.<#= FormatEnum(this.Model.Role, RoleMap) #>, [.. this.GetContentAsync(context).ToEnumerable()]) { AdditionalProperties = this.GetMetadata() };
```

Comment:
While I'm sorry my removal of System.Linq.Async broke you, I'm glad it brought to light this sync-over-async naughtiness ;-)